dist/profile/manifests/pkgrepo.pp

```diff
@@ -117,22 +114,39 @@
     error_log_pipe  => "|/usr/bin/rotatelogs -t ${apache_log_dir}/error_nonssl.log.%Y%m%d%H%M%S 604800",
   }
 
-  apache::vhost { 'pkg.jenkins-ci.org':
+  apache::vhost { 'pkg.jenkins-ci.org unsecured':
+    servername      => 'pkg.jenkins-ci.org',
     port            => 80,
     docroot         => $docroot,
-    override        => ['All'],
-    options         => 'Indexes FollowSymLinks MultiViews',
 
     access_log_pipe => "|/usr/bin/rotatelogs -t ${apache_log_dir}/access_legacy_nonssl.log.%Y%m%d%H%M%S 604800",
     error_log_pipe  => "|/usr/bin/rotatelogs -t ${apache_log_dir}/error_legacy_nonssl.log.%Y%m%d%H%M%S 604800",
-    require         => Apache::Vhost[$repo_fqdn],
+    redirect_dest   => ['https://pkg.jenkins.io'],
+  }
+
+  apache::vhost { 'pkg.jenkins-ci.org':
+    servername      => 'pkg.jenkins-ci.org',
+    port            => 443,
+    docroot         => $docroot,
+    ssl             => true,
+
+    access_log_pipe => "|/usr/bin/rotatelogs -t ${apache_log_dir}/access_legacy.log.%Y%m%d%H%M%S 604800",
+    error_log_pipe  => "|/usr/bin/rotatelogs -t ${apache_log_dir}/error_legacy.log.%Y%m%d%H%M%S 604800",
+    require         => Apache::Vhost['pkg.jenkins-ci.org'],
```

Comment:
```suggestion
```

Because there is no reason to depend on another vhost here

Reply:
Side note for improvement: All the apache vhosts should require `File[$docroot]` because... the doc root is required. Requiring `File[$docroot]` transitively requires `File[$apache_log_dir]` which is perfect

Reply:
PR rebases + this proposed change was applied